fabtests/ubertest/uber.c

```diff
@@ -752,5 +752,13 @@ int main(int argc, char **argv)
 	if (opts.dst_addr)
 		fts_close(series);
 	ft_free();
+
+	if (results[FT_ENOSYS])
+		ret = -FI_ENOSYS;
+	else if (results[FT_EIO])
+		ret = -FI_EIO;
```

Comment:
EIO represent real errors, please rank the highest.  ENOSYS could just indicate some feature isn't supported on the current platform.